hardhat.config.js

```diff
@@ -70,7 +70,7 @@ module.exports = {
   solidity: {
     compilers: [
       {
-        version: "0.8.4",
+        version: "0.8.11",
```

Comment:
you bumped this version to 0.8.11 but all the contracts are set to 0.8.4. is this intentional?

Reply:
Yup. Contracts specify a minimum version, not a required version.